tactical-microgrid-standard/cli/CLI.mpc

```diff
@@ -0,0 +1,18 @@
+project: dcpsexe {
+  after += Commands_Idl
+  exename = cli
+  includes += ..
+  libpaths += ../idl ../common ../cli_idl
+  libs += tms Commands_Idl Common
+
+  IDL_Files {
+  }
+
+  Typesupport_Files {
+  }
+
+  Source_Files {
+    main.cpp
+    CLIClient.cpp
+  }
```

Comment:
It's hard to tell just from looking at the diff, but if these are the only `*.cpp` files in the directory, you can just let it default the source files.  

If there are no `*.idl` files in the directory, the scopes above for IDL_Files and Typesupport_Files can be removed.